metricbeat/module/ceph/pool_disk/testdata/sample_response.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
If we move this file to the module level under `_meta/testdata` we don't have to duplicate it and both metricsets can reuse it.